src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1279,14 +1279,18 @@ internal override IEnumerable<FieldSymbol> GetFieldsToEmit()
                 switch (m.Kind)
                 {
                     case SymbolKind.Field:
-                        var field = (FieldSymbol)m;
-                        yield return field.TupleUnderlyingField ?? field;
+                        if (m is TupleErrorFieldSymbol)
```

Comment:
>if (m is TupleErrorFieldSymbol) [](start = 24, length = 31)

When do we try to emit with error fields?

Reply:
```
struct ValueTuple<T1, T2> { }
```

---
In reply to: [425348604](https://github.com/dotnet/roslyn/pull/44231#discussion_r425348604) [](ancestors = 425348604)